crates/liboci-cli/src/features.rs

```diff
@@ -6,4 +6,61 @@ use clap::Parser;
 /// It is documented here:
 /// https://github.com/opencontainers/runtime-spec/blob/main/features-linux.md
 #[derive(Parser, Debug)]
-pub struct Features {}
+pub struct Features {
```

Comment:
Hey, so this struct is for taking in the `feature` command options, not the one for storing the actual data related to features. the errors are given because clap cannot derive the field of this. I'd suggest that keep this structure as-is, and make a new one, which has all the fields that you have added here. You'll need to derive a serde json for it, so that we can output it as a json string.